app/controller/dashboard.js

```diff
@@ -1,75 +1,63 @@
+// Imports for packages
 import Hubkit from 'hubkit';
 import Promise from 'bluebird';
 import _ from 'lodash';
 import ini from 'ini';
 
+// Local Imports
 import app from '~/';
 import { isBeta } from './auth';
 import { Application } from '~/model/application';
 
-// Create Hubkit with a static instance
-let gh = new Hubkit();
+// Global instances
+const gh = Promise.promisifyAll(new Hubkit());
```

Comment:
If I remember correctly it was something with .map or .filter, but it looks like I changed what I was working on. I'll change it back.